stories/core_components/GeoPushpinChart.tsx

```diff
@@ -56,6 +56,18 @@ storiesOf("Core components/GeoPushpinChart", module)
             }),
         ),
     )
+    .add("with location and zoom config", () =>
```

Comment:
I suggest `with location and show unclustered pins` and `zoom: 5` is fine which could display more pins :)

Reply:
with `zoom: 5`, while panning the map to the left, you will see a clustered point.
IMO, `zoom: 6` is better

Reply:
updated